### ISSUE TITLE
Not supported on mobile?

### ISSUE THREAD

**Issue by tdmalone** [apsillers/stacklife#3]
Just noting that this might not be supported on mobile (I haven't taken a look on desktop yet).

When clicking any of the buttons from within your profile on the StackExchange iOS app, the following error shows:

> TypeError: Parameter 'url' must be a string, not undefined at Url.parse (url.js:107:11) at Object.urlParse [as parse](url.js:101:5) at postActionRedirect (/home/mils/solife/lifeserver.js:21:35) at Object.grid [as handle](/home/mils/solife/lifeserver.js:100:9) at next (/home/mils/solife/node_modules/express/node_modules/connect/lib/proto.js:193:15) at pass (/home/mils/solife/node_modules/express/lib/router/index.js:110:24) at Router._dispatch (/home/mils/solife/node_modules/express/lib/router/index.js:173:5) at Object.router (/home/mils/solife/node_modules/express/lib/router/index.js:33:10) at next (/home/mils/solife/node_modules/express/node_modules/connect/lib/proto.js:193:15) at Object.expressInit [as handle](/home/mils/solife/node_modules/express/lib/middleware.js:30:5) at next (/home/mils/solife/node_modules/express/node_modules/connect/lib/proto.js:193:15)

**Comment by apsillers** [comment]
Interesting! I can also repro this in the Android SE app. Works fine in Android Chrome.

I will check out what the SE app is doing differently. Thanks!

**Comment by apsillers** [comment]
@tdmalone Over a year later, I finally solved this one! Turns out I had some unpushed changes in my server that use the `Referer` header to decide whether to redirect to HTTP or HTTPS. The Stack Exchange app doesn't send a `Referer` header when you visit (unsurprisingly, since the visit isn't coming from a previous website at all).

I honestly forget why this mess was in there in the first place, but since (1) Stack Overflow is now all HTTPS and (2) Let's Encrypt has made HTTPS certs free, I can just go all-in on HTTPS and get rid of that stuff anyway. Thanks for reporting!
